src/app/client/src/app/modules/core/services/org-details/org-details.service.ts

```diff
@@ -126,6 +126,25 @@ export class OrgDetailsService {
     return this.learnerService.get(systemSetting);
   }
 
+  getCommingSoonMessage() {
+    const systemSetting = {
```

Comment:
put this inside else case

Reply:
const systemSetting = {... inside else case as this will be used to make api call